Sources/VLCMovieViewController.m

```diff
@@ -1234,39 +1295,65 @@ - (IBAction)sleepTimerAction:(id)sender
+            if (UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation])) {
+                _multiSelectionView.showsEqualizer = YES;
+            } else {
+                _multiSelectionView.showsEqualizer = NO;
+            }
         }
-    }
 
-    CGRect workFrame = _multiSelectionView.frame;
-    workFrame.size = [_multiSelectionView proposedDisplaySize];
-    workFrame.origin.x = CGRectGetMaxX(sender.frame) - workFrame.size.width;
+        CGRect workFrame = _multiSelectionView.frame;
+        workFrame.size = [_multiSelectionView proposedDisplaySize];
+        workFrame.origin.x = CGRectGetMaxX(sender.frame) - workFrame.size.width;
 
-    _multiSelectionView.alpha = 1.0f;
+        _multiSelectionView.alpha = 1.0f;
 
-    /* animate */
-    _multiSelectionView.frame = CGRectMake(workFrame.origin.x, workFrame.origin.y + workFrame.size.height, workFrame.size.width, 0.);
-    [UIView animateWithDuration:.3
-                     animations:^{
-                         self->_multiSelectionView.frame = workFrame;
-                         self->_multiSelectionView.hidden = NO;
-                     }
-                     completion:^(BOOL finished){
-                     }];
+        /* animate */
+        _multiSelectionView.frame = CGRectMake(workFrame.origin.x, workFrame.origin.y + workFrame.size.height, workFrame.size.width, 0.);
+        [UIView animateWithDuration:.3
+                         animations:^{
+                             self->_multiSelectionView.frame = workFrame;
+                             self->_multiSelectionView.hidden = NO;
+                         }
+                         completion:^(BOOL finished){
+                         }];
+        #else
+            if (_videoOptionsControlBar.hidden == NO) {
+                [UIView animateWithDuration:.3
+                                 animations:^{
+                                     self->_videoOptionsControlBar.hidden = YES;
+                                 }
+                                 completion:^(BOOL finished){
+                                 }];
+                return;
+            }
+            CGRect workFrame = _videoOptionsControlBar.frame;
+            workFrame.origin.x = CGRectGetMaxX(sender.frame) - workFrame.size.width;
+    
+            _videoOptionsControlBar.alpha = 1.0f;
+    
+            /* animate */
+           _videoOptionsControlBar.frame = CGRectMake(workFrame.origin.x, workFrame.origin.y + workFrame.size.height, workFrame.size.width, 0.);
+            [UIView animateWithDuration:.3
+                             animations:^{
+                                 self->_videoOptionsControlBar.frame = workFrame;
+                                 self->_videoOptionsControlBar.hidden = NO;
+                             }
+                             completion:^(BOOL finished){
+                             }];
+        #endif
```

Comment:
in general I would move this code into a function instead of having this big if else
```
#if !new UI
 [self layoutframeformultiselection]
#else 
[self layoutframeforVideooptions]
#endif
```
 makes it way more readible